qemu/tests/cfg/hpt_max_page_size.cfg

```diff
@@ -0,0 +1,28 @@
+- hpt_max_page_size: install setup image_copy unattended_install
+    type = hpt_max_page_size
+    only ppc64 ppc64le
+    only Linux
+    only virtio_scsi
+    virt_test_type = qemu
+    required_qemu = [4.0.0,)
+    kill_vm = yes
+    start_vm = no
+    pre_command_noncritical = yes
+    setup_hugepages = yes
+    hp_size_cmd = cat /proc/meminfo|grep Huge | grep -E '*HugePages:' | awk '{print $2}'
+    machine_type_extra_params = max-cpu-compat=power8
+    variants:
+        - 64k:
+            machine_type_extra_params += ,cap-hpt-max-page-size=64k
+            variants:
+                - without_men_backing_file:
```

Comment:
typo: men_backing --> mem_backing
However, we can launch guest using `-mem-path /mnt/kvm_hugepage` does not mean it uses hugepages, so this variants it not suitable.
```bash
$ mount | grep kvm_hugepage

$ /usr/libexec/qemu-kvm -mem-path /mnt/kvm_hugepage
VNC server running on ::1:5900
```

Reply:
thanks a lot, change it.